cloud

```diff
@@ -1243,6 +1246,23 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r bucket="$1"; shift
+    TMPDIR="$TMPDIR_ORIG" gsutil -q mb -p "$project" -c STANDARD -l \
+        "us-central1" -b on "gs://$bucket"
+    TMPDIR="$TMPDIR_ORIG" gsutil iam ch allUsers:objectViewer "gs://$bucket/"
+}
+
+# Remove a Google Cloud Storage Bucket and its contents
+# Args: bucket
+function storage_withdraw() {
+    declare -r bucket="$1"; shift
+    TMPDIR="$TMPDIR_ORIG" gsutil -q -m rm -r "gs://$bucket"
```

Comment:
Why do you use the `-m` option here?

Reply:
Without it the object of the bucket will get delete one by one but `-m` parallel deletions can be done which can make process faster of deletion, [Supported Link](https://cloud.google.com/storage/docs/gsutil/addlhelp/GlobalCommandLineOptions)

Reply:
okay